internal/provider/utils_test.go

```diff
@@ -195,3 +196,33 @@ func Test_sliceDiff_model(t *testing.T) {
 		})
 	}
 }
+
+func Test_newNameOrIdList(t *testing.T) {
+	tests := []struct {
+		name string
+		args []attr.Value
+		want []oxide.NameOrId
+	}{
+		{
+			name: "success double quote",
+			args: []attr.Value{types.StringValue("d72b0406-dd16-4373-ba2e-b7016ede6c3c"), types.StringValue("bob")},
+			want: []oxide.NameOrId{"d72b0406-dd16-4373-ba2e-b7016ede6c3c", "bob"},
+		},
+		{
+			name: "success backtick",
+			args: []attr.Value{types.StringValue(`d72b0406-dd16-4373-ba2e-b7016ede6c3c`), types.StringValue(`bob`)},
+			want: []oxide.NameOrId{"d72b0406-dd16-4373-ba2e-b7016ede6c3c", "bob"},
+		},
+	}
```

Comment:
I was not successful in mocking a failure here, so just testing success cases 😅

Reply:
Thank you Karen!